libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts

```diff
@@ -212,52 +202,22 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
   }
 
   #affixer: SkyAffixer | undefined;
-
-  #ngUnsubscribe = new Subject<void>();
-
+  #affixService = inject(SkyAffixService);
+  #changeDetector = inject(ChangeDetectorRef);
   #overlay: SkyOverlayInstance | undefined;
-
-  #_buttonStyle = 'default';
-
-  #_buttonType: SkyDropdownButtonType | string = DEFAULT_BUTTON_TYPE;
-
+  #ngUnsubscribe = new Subject<void>();
+  #overlayService = inject(SkyOverlayService);
+  #positionTimeout: number | undefined;
+  #themeSvc = inject(SkyThemeService, { optional: true });
+  #zIndex = inject(SKY_STACKING_CONTEXT, { optional: true })?.zIndex;
```

Comment:
Let's group the `inject` properties at the bottom of the list, nearest to the `constructor`.

Reply:
updated